### PR TITLE
Use yarn

### PR DESCRIPTION
**Pull request by adi**
This is possible implementation for the request in #2651

### CODE REVIEW

**Comment by adi** [comment]
I signed it!

**Comment by Brocco** [comment]
This will need to have tests associated to get in. And I want to allow the discussion to continue further in issue #2651 to allow others to weigh in.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (filipesilva) [inline package.json]: Please revert all changes to this file.

Package name should not change, and the version will be updated via the release process.

The two extra dependencies added are subpackages of this one, and are already added in the appropriate `package.json`.

File: packages/angular-cli/commands/init.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import LinkCli from '../tasks/link-cli';
 import NpmInstall from '../tasks/npm-install';
+import Yarn from '../tasks/yarn';
```
Comment (filipesilva) [inline packages/angular-cli/commands/init.ts]: Please name task `YarnInstall` and file `yarn-install`

**Comment by adi** [comment]
Considering --skip-npm and other possible solutions for running yarn with angular-cli, this PR is not as important as I thought intially.

**Comment by mgol** [comment]
Just a note - both Angular 1 & 2 as well as other Angular projects are
switching their dev systems from npm to yarn. I understand we can't require
user projects to do that, though.

**Pull request closed without merging**